packages/@react-spectrum/list/test/ListView.test.js

```diff
@@ -236,12 +283,14 @@ describe('ListView', function () {
         });
 
         it('should move focus to previous cell in RTL', function () {
-          let tree = renderListWithFocusables('ar-AE');
+          let tree = renderListWithFocusables({locale: 'ar-AE'});
           // Should move from button two to button one
-          let start = within(tree.getAllByRole('row')[0]).getAllByRole('button')[1];
-          let end = within(tree.getAllByRole('row')[0]).getAllByRole('button')[0];
-          act(() => start.focus());
+          let start = within(getRow(tree, 'Foo')).getAllByRole('button')[1];
+          let end = within(getRow(tree, 'Foo')).getAllByRole('button')[0];
+          // Need to press to set a modality, otherwise useSelectableCollection will think this is a tab operation
```

Comment:
have to wonder if `.focus` is the best way to do it anyways, I think it might make more sense to call `userEvent.tab()`, that'd be more representative of the real world

Reply:
for the other instances, not for this one, where clicking is the real life thing happening